qiskit/visualization/dag_visualization.py

```diff
@@ -15,89 +15,22 @@
 """
 Visualization function for DAG circuit representation.
 """
-
-import os
-import sys
-import tempfile
+from retworkx.visualization import graphviz_draw
 
 from qiskit.dagcircuit.dagnode import DAGOpNode, DAGInNode, DAGOutNode
 from qiskit.circuit import Qubit
 from qiskit.utils import optionals as _optionals
 from qiskit.exceptions import InvalidFileError
 from .exceptions import VisualizationError
 
-FILENAME_EXTENSIONS = {
-    "bmp",
-    "canon",
-    "cgimage",
-    "cmap",
-    "cmapx",
-    "cmapx_np",
-    "dot",
-    "dot_json",
-    "eps",
-    "exr",
-    "fig",
-    "gd",
-    "gd2",
-    "gif",
-    "gv",
-    "icns",
-    "ico",
-    "imap",
-    "imap_np",
-    "ismap",
-    "jp2",
-    "jpe",
-    "jpeg",
-    "jpg",
-    "json",
-    "json0",
-    "mp",
-    "pct",
-    "pdf",
-    "pic",
-    "pict",
-    "plain",
-    "plain-ext",
-    "png",
-    "pov",
-    "ps",
-    "ps2",
-    "psd",
-    "sgi",
-    "svg",
-    "svgz",
-    "tga",
-    "tif",
-    "tiff",
-    "tk",
-    "vdx",
-    "vml",
-    "vmlz",
-    "vrml",
-    "wbmp",
-    "webp",
-    "xdot",
-    "xdot1.2",
-    "xdot1.4",
-    "xdot_json",
-}
-
 
-@_optionals.HAS_PYDOT.require_in_call
+@_optionals.HAS_GRAPHVIZ.require_in_call
 def dag_drawer(dag, scale=0.7, filename=None, style="color"):
     """Plot the directed acyclic graph (dag) to represent operation dependencies
     in a quantum circuit.
 
-    Note this function leverages
-    `pydot <https://github.com/erocarrera/pydot>`_ to generate the graph, which
-    means that having `Graphviz <https://www.graphviz.org/>`_ installed on your
-    system is required for this to work.
-
-    The current release of Graphviz can be downloaded here: <https://graphviz.gitlab.io/download/>.
-    Download the version of the software that matches your environment and follow the instructions
-    to install Graph Visualization Software (Graphviz) on your operating system.
+    This function calls the `graphviz_draw` function from the `retworkx`
```

Comment:
This will be rendered as _graphviz_draw_ in the output docs. You should use:
```
``graphviz_draw``
````
 to get it to render like `graphviz_draw` in the output. But that being said I think you can just do
```
:func:`~retworkx.visualization.graphviz_draw`
```

and we can add the [intersphinx](https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html) configuration to `docs/conf.py` (either in this PR or in a follow up) to enable linking directly to the retworkx docs using the reference markup.

Reply:
Thanks for the suggestion. I've made the necessary changes and verified by generating the docs locally